projects/graph/social/social.py

```diff
@@ -73,12 +72,38 @@ def getAllSocialPaths(self, userID):
         """
         visited = {}  # Note that this is a dictionary, not a set
         # !!!! IMPLEMENT ME
+        for i in range(1, len(self.users) + 1):
+            if len(self.friendships[i]) is not 0:
+                visited[i] = self.bft_path(userID, i)
+                print(visited)
+            
         return visited
 
 
+    def bft_path(self, start, target):
+        from queue import Queue
+        if start in self.friendships:
+            nextItems = Queue()
+            visited = []
+            path = []
+            nextItems.put([start])
+            while nextItems.empty() is not True:
+                first = nextItems.get()
```

Comment:
bugs can be pretty redundant to your code and I'm glad you went back and fixed your issue